libcodechecker/libhandlers/server.py

```diff
@@ -129,7 +129,7 @@ def add_arguments_to_parser(parser):
     parser.add_argument('--skip-db-cleanup',
                         dest="skip_db_cleanup",
                         action='store_true',
-                        default=True,
+                        default=False,
```

Comment:
For boolean values, I'd rather have `argparse.SUPPRESS` as default value. No need to have a `(default: False)` in the help if it's a toggle.

Reply:
You enabled the database cleanup on server start by default. In this case it would be better to create an index on the `file_id` column of the `reports` table? (See: #1130) It can be created in a separate pull request.

Reply:
I've added the schema modification to this pull request in a separate commit.